src/MyClass.java

```diff
@@ -1,10 +1,13 @@
-class myClass {
-    int Year = 2023;
-    public static int MYMETHOD(int a,
-                               int b){
-        int c = a + b; //sum of 2 numbers
-        return c;}
-    public static void main(String[]
-                                    args) {
-        MYMETHOD(num1, num2);}
-}
+class MyClass {
+
+    public static int myMethod(int a, int b) {
+        return a + b;
+    }
+
+    public static void main(String[] args) {
+        int num1 = 5;
+        int num2 = 7;
+        int result = myMethod(num1, num2);
```

Comment:
it's ok but try to write as low as you can -  System.out.println(sumOfTwoNumbers(5, 7));